wysadzulice/assets/js/new_campaign.js

```diff
@@ -0,0 +1,40 @@
+import jquery from 'jquery';
+import Planting from 'plantingjs';
+import { View } from 'backbone';
+
+
+export default View.extend({
+
+  initialize: function(options) {
+    this.$el = jquery('.viewport');
+    this.manifestoUrl = options.manifestoUrl;
+    this.googleApiKey = options.googleApiKey;
+    this.saveUrl = options.saveUrl;
+    this.render();
```

Comment:
What this view does is creating new instance of Planting engine. I wouldn't call it rendering since this class is not responsible for generate html.